### PR TITLE
fixed feature type 1 - Point

### PR DESCRIPTION
**Pull request by juloaiza**
For feature type 1 (point) the module raise an error. Bug fixed

### CODE REVIEW

File: vt2pbf/service/feature.py [hunk 1]
```diff
@@ -78,6 +78,8 @@ def _encode_feature_geometry(self, raw_geometry: List[List[List[int]]]) -> List[
         x = 0
         y = 0
         result = []
+        # handle point feature type
+        raw_geometry = [raw_geometry] if self.feature_type == 1 else raw_geometry
```
Comment (DenysMoskalenko) [inline vt2pbf/service/feature.py:82]: According to typehint in this function raw_geometry must have a structure like "List[List[List[int]]]"
It means you cannot just wrap raw_geometry once more cause of feature_type.

I sure the problem should be resolved before this function, namely in add_geometry. Also add_geometry typehints must be changed for more basic structure to support Poin geometry

**Review by DenysMoskalenko — CHANGES_REQUESTED** [review]

File: vt2pbf/service/feature.py (hunk 1 above)
Comment (DenysMoskalenko) [reply vt2pbf/service/feature.py:82]: I u want u can make this changes and I will merge PR and update library version, if not - I will do it sometimes

**Comment by DenysMoskalenko** [comment]
Look on you fork PR https://github.com/juloaiza/vt2pbf/pull/1

**Review by DenysMoskalenko — APPROVED** [review]

**Comment by juloaiza** [comment]
Thank you!

**Pull request merged**